app/scripts/views/openid.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'views/base',
+  'stache!templates/openid',
+  'lib/session'
+],
+function (Cocktail, BaseView, Template, Session) {
+  'use strict';
+
+  var View = BaseView.extend({
+    template: Template,
+    className: 'openid-sign-in',
+
+    initialize: function (options) {
+      BaseView.prototype.initialize.call(this, options);
+      options = options || {};
+
+      this._err = this.relier.get('err');
+      this._uid = this.relier.get('uid');
+      this._sessionToken = this.relier.get('session');
+      this._keyFetchToken = this.relier.get('key');
+      this._unwrapBKey = this.relier.get('unwrap');
```

Comment:
>  Could we use localStorage or similar rather than passing it in the URL?

N/m, I see from https://github.com/mozilla/fxa-auth-server/pull/1007 that is has to be passed in the URL from the auth-server.  I'll add more comments over there.